### ISSUE TITLE
Player markers (admin menu) can't be disabled

### ISSUE THREAD

**Issue by Jawshy** [ArmaLife/Framework#365]
Player markers enabled in the admin menu cannot be disabled. Nothing happens when clicking the "Markers" button again. No errors in any log file. 

Relevant file(s): Altis/Altis_Life.Altis/core/admin/[fn_adminMarkers.sqf](https://github.com/ArmaLife/Altis/blob/master/Altis_Life.Altis/core/admin/fn_adminMarkers.sqf)

Game version: 1.56.134627
Mission version: v4.4r1 - latest commit: 201f526972d789caabe38ebc41e01bb4624cacfb

**Comment by HyunsukBaek** [comment]
i have this issues too, when A user already disconnected, but market has not disabled.

**Comment by danielstuart14** [comment]
hmmmmm, that's weird... I'll take a look on it, but, even the hint doesn't show up?

**Comment by Jawshy** [comment]
Correct, even the [STR_ANOTF_MDisabled](https://github.com/ArmaLife/Altis/blob/master/Altis_Life.Altis/stringtable.xml#L468) hint does not show.

**Comment by trinitynet** [comment]
#407 Plz test thankies

**Comment by trinitynet** [comment]
scrap that, still dont work properly

**Comment by tkcjesse** [comment]
@trinitynet do you want #407 closed?

**Comment by trinitynet** [comment]
plz

**Comment by NiiRoZz** [comment]
6ee59925d6ce458aab48c66b9bb64632f55ce6f2

**Comment by Jawshy** [comment]
This is an issue again. Player markers cannot be disabled, but the disabled hint will display. No errors in RPT files. 

Game version: [1.58.135742](https://dev.arma3.com/post/spotrep-00055)
Mission version: v4.4r2 as of https://github.com/ArmaLife/Framework/commit/e337b75845ba2b5cf387176ff2ccfcfa9933c20e

**Comment by HyunsukBaek** [comment]
![20160518233857_1](https://cloud.githubusercontent.com/assets/5819861/15362934/e09bc508-1d51-11e6-8961-113cde0313a9.jpg)

and this issue too, ERROR displayed.

**Comment by BoGuu** [comment]
All this code needs to change, frankly, it's not great.

https://github.com/ArmaLife/Framework/blob/master/life_server/functions.sqf#L311

**Comment by HyunsukBaek** [comment]
is this issue fixed?

**Comment by Jawshy** [comment]
No, it will be fixed in pull request https://github.com/ArmaLife/Framework/pull/748 once and if it is merged.
